modules/release-notes/pages/sdk-release-notes.adoc

```diff
@@ -10,7 +10,7 @@
 :page-layout: releasenotes
 :sdk-short-name: DFINITY Canister SDK
 :sdk-long-name: DFINITY Canister Software Development Kit (SDK)
-:release: 0.5.3
+:release: 0.5.4
```

Comment:
Perhaps we should keep track of all past releases?

Also can we add these notes in Github too?

Reply:
they are included in the tagged archives for doc (so 0.5.3 doc archive includes the release notes). I'll add the steps to the shepherd and process docs.